src/g_game.c

```diff
@@ -2189,6 +2192,11 @@ static void G_DoLoadGame(void)
   // done
   Z_Free(savebuffer);
 
+  // [Alaux] Update smooth count values;
```

Comment:
Is this even required? I mean, `G_DoLoadGame()` calls calls `G_InitNew()` a few lines earlier which in turn calls `G_DoLoadLevel()` which already has this code.

Reply:
If I remember correctly, at least at the time I wrote the code originally, `G_DoLoadLevel()` by itself would load a different value than the one loaded by `G_DoLoadGame()`; like, the game would first load a base level (health = 100; armor = 0) and then change it based on the savegame data (potentially different health/armor values). Whatever the technical explanation, in practice having both updates made it so that the smooth count values would instantly change to the logical health value both when loading a new game and/or a savegame.

Reply:
Yes, you are right. If I remove these lines, then the savegame will load with the correct health/armor values, but they will be "smoothed" to starting from the init values.

Reply:
Yep, said smoothing is what I was trying to prevent. Granted, it could be an intentional effect (could even start at 0, I think Heretic does that?), but for my implementation I wanted to keep it quick.

Reply:
Yes, right, let's keep it as you did it.